### ISSUE TITLE
Create a file with a list of Contributors

### ISSUE THREAD

**Issue by agriffard** [OrchardCMS/OrchardCore.Translations#30]
List the most active contributors on Crowdin per language.

If a translator wants to appear in the list, please make a PR when the file is available or add a comment with your Full Name, Github Handle and Language.

**Comment by hishamco** [comment]
May I help to assist you for creating this list

**Comment by agriffard** [comment]
Thank you @hishamco . 

A Contributing.md could also be useful to explain how to translate strings on Crowdin.

**Comment by hishamco** [comment]
> A Contributing.md could also be useful to explain how to translate strings on Crowdin.

You are right, we can spend time to it, thanks

**Comment by hishamco** [comment]
@neman please let us now if you 're interested in Serbian translations

@agriffard any updates on CONTRBUITING.md?

**Comment by neman** [comment]
Yes, I am. At his moment our team (me included) us busy implementing OC on one egovernment project, but as soon as I free up, I will translate it. Di I have to signup on Crowdin?

**Comment by agriffard** [comment]
Contributors added to the main repo with a _translation_ contribution.

**Comment by hishamco** [comment]
Nah, I really forgot that we already added a little icon for translators ;)
